### ISSUE TITLE
how to use mouseJoint without the testbed

### ISSUE THREAD

**Issue by bobcgausa** [shakiba/planck.js#178]
I'm writing a p5 interface for planck and i don't use the testbed because i want p5 graphics.
however i can't create a mouse joint in mousePressed

so i created a mousejoint statically

now what can i do in mousePressed to set the target body and location?
any help appreciated

here's the example
[](https://editor.p5js.org/Bobcook47/sketches/OlzHvRszI)

i only want to drag the figure that i click

**Comment by bobcgausa** [comment]
here's the p5 example
https://editor.p5js.org/Bobcook47/sketches/OlzHvRszI

**Comment by shakiba** [comment]
Have you checked out testbed code? It is in testbed folder.

**Comment by bobcgausa** [comment]
Yes I looked at testbed codeI have to create mousejoint in setupThen in mousepressed I get body and need to reset body a and b.As code shows, it didn't workTestbed doesn't work that wayOn Apr 21, 2021 2:34 AM, Ali Shakiba ***@***.***> wrote:
Have you checked out testbed code? It is in testbed folder.
